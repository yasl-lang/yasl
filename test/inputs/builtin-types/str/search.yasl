echo 'YASL'->search('A')
echo 'YASL'->search('0')

echo 'ABC ABC'->search('AB', 1)

echo 'YASL'->has('A')
echo 'YASL'->has('0')

echo 'ABC ABC'->has('AB', 2)
