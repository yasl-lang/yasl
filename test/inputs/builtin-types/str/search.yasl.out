1, 2
undef
4, 6
true
false
true
